src/components/SchemaTable/index.jsx

```diff
@@ -36,12 +36,19 @@ const useStyles = makeStyles(theme => ({
   keywordColumn: {},
   descriptionColumn: {},
   line: {
+    display: 'flex',
+    alignItems: 'center',
```

Comment:
It's centered on Chrome but on Firefox, it looks a bit off:

<img width="190" alt="Screen Shot 2019-12-27 at 8 59 14 AM" src="https://user-images.githubusercontent.com/3766511/71519864-45810200-2887-11ea-87e3-e89e62d759f5.png">

Reply:
Applied changes! Hope this is better 😃 

| Chrome |  Firefox |
|:---|:---|
| <img src="https://user-images.githubusercontent.com/29671309/71574777-9ca80080-2b2d-11ea-9207-b7113f639f1e.png" width="170px" /> | <img src="https://user-images.githubusercontent.com/29671309/71574734-5fdc0980-2b2d-11ea-8bae-647092e66679.png" width="170px" /> |